benefits/static/css/styles.css

```diff
@@ -375,6 +375,8 @@ footer .footer-links li a.footer-link:visited {
 
 .btn.btn-lg.btn-primary:focus,
 .btn.btn-lg.btn-primary:focus-visible,
+.btn.btn-lg:focus,
+.btn.btn-lg:focus-visible,
```

Comment:
This is for the Login.gov Sign Out button on the Enrollment Success page for the Login.gov flows.